spec/architecture-decision-records/1681149688-remove-d1-completely.md

```diff
@@ -0,0 +1,122 @@
+# stop using d1 with `@web3-storage/access-api`
+
+<!--
+This is a minimal template. Feel free to add more sections as needed.
+
+Please review also the Design Doc template and add any relevant sections to your ADR:
+https://www.notion.so/pl-strflt/Writing-a-Design-Doc-aa6034be43c2434ba88a2fd844516e94
+-->
+
+> Status: PROPOSED
+
+Authors
+* @gobengo
+* @travis
+
+<!--
+PROPOSED, ACCEPTED, REJECTED, DEPRECATED, SUPERSEDED BY {link-to-ADR}
+-->
+
+## Context
+
+<!--
+What is the issue that we're seeing that motivates this decision or change?
+-->
+
+### Prior Documents
+
+* [`@web3-storage/access-api` data storage](https://hackmd.io/f5PiUFarQJiq6L0K7xaLQA)
```

Comment:
explain what this contains and how it's related (documents how access-api uses d1 today)